examples/chart/teleport-kube-agent/templates/updater/deployment.yaml

```diff
@@ -67,6 +67,9 @@ spec:
           - "--base-image={{ include "teleport-kube-agent.baseImage" . }}"
           - "--version-server={{ tpl $updater.versionServer . }}"
           - "--version-channel={{ $updater.releaseChannel }}"
+          {{- if .Values.updater.extraArgs }}
+            {{- toYaml .Values.updater.extraArgs | nindent 10 }}
```

Comment:
can we add a snapshot to ensure the indentation is always ok?
I am now paranoid about that

Reply:
If indentation is not OK the `contains` test will properly fail as the item won't be in the list. The issue earlier was caused by the fact we did 2 rounds of dumping yaml/parsing yaml because of the config merge hack.

Reply:
oh you are correct